# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rocblas/hotfix-libdir.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rocBLAS-rocm-6.4.0/library/src/CMakeLists.txt.vanilla	2025-04-12 18:48:50.488443107 +0200
+++ rocBLAS-rocm-6.4.0/library/src/CMakeLists.txt	2025-04-12 18:49:25.388258995 +0200
@@ -682,7 +682,7 @@
 
 rocblas_library_settings( rocblas )
 
-target_link_libraries( rocblas PRIVATE "-Xlinker --exclude-libs=ALL" ) # HIDE symbols
+target_link_libraries( rocblas PRIVATE "-Xlinker --exclude-libs=ALL -L/opt/rocm/lib64 -L/opt/rocm/lib" ) # HIDE symbols
 
 if(BUILD_WITH_HIPBLASLT)
   find_package( hipblaslt ${HIPBLASLT_VERSION} REQUIRED CONFIG PATHS ${hipblaslt_path} ${ROCM_PATH})
